### PR TITLE
Removed references for js templates

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # BLUI-3642

#### Changes proposed in this Pull Request:
- Removed js templates React and React Native 

#### To Test:
- Please refer to testing locally instructions in Readme
- npx -p @brightlayer-ui/cli blui new react-native --cli=rnc --template=blank-typescript
Templates available now:
blank-typescript
routing-typescript
authentication-typescript

**Note: DO NOT MERGE until https://github.com/brightlayer-ui/brightlayer-ui-cli/pull/144 gets merge**

_**In future, we can rename the templates from blank-typescript to blank, but for now lets use the same**_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,37 +5,47 @@ These are project templates that are used by the Brightlayer UI CLI when startin
 ## Available Templates
 
 ### Blank
-The Blank template provides a bare-bones application with Brightlayer UI components and themes automatically configured and ready to use. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-blank-typescript)(recommended) and [JavaScript](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-blank).
+
+The Blank template provides a bare-bones application with Brightlayer UI components and themes automatically configured and ready to use. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-blank-typescript).
```
Comment (daileytj) [inline README.md:9]:
```suggestion
The [Blank template](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-blank-typescript) provides a bare-bones application with Brightlayer UI components and themes automatically configured and ready to use.
```

File: README.md [hunk 2]
```diff
@@ -5,37 +5,47 @@ These are project templates that are used by the Brightlayer UI CLI when startin
...
 
 ### Routing
-The Routing template provides all of the essential Brightlayer UI configuration as well as additional configuration for using [React Navigation](https://reactnavigation.org/). It also adds a navigation Drawer to switch between several sample screens. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-routing-typescript)(recommended) and [JavaScript](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-routing).
+
+The Routing template provides all of the essential Brightlayer UI configuration as well as additional configuration for using [React Navigation](https://reactnavigation.org/). It also adds a navigation Drawer to switch between several sample screens. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-routing-typescript).
```
Comment (daileytj) [inline README.md:13]:
```suggestion
The [Routing template](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-routing-typescript) provides all of the essential Brightlayer UI configuration as well as additional configuration for using [React Navigation](https://reactnavigation.org/). It also adds a navigation Drawer to switch between several sample screens.
```

File: README.md [hunk 3]
```diff
@@ -5,37 +5,47 @@ These are project templates that are used by the Brightlayer UI CLI when startin
...
 
-|                | placeholder landing page | navigation drawer & routing  | login & registration screens  |
-| -------------- | -- | -- | -- |
-| Blank          | ✅ |    |    |
-| Routing        | ✅ | ✅ |    |
-| Authentication | ✅ | ✅ | ✅ |
+The Authentication template includes everything from the Routing template and also installs and configures the Brightlayer UI [react-native-auth-workflow](https://www.npmjs.com/package/@brightlayer-ui/react-native-auth-workflow) package to automatically wrap the main application with functions and screens for Login, Registration, Change Password, etc. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-authentication-typescript).
```
Comment (daileytj) [inline README.md:17]:
```suggestion
The [Authentication template](https://www.npmjs.com/package/@brightlayer-ui/react-native-template-authentication-typescript) includes everything from the Routing template and also installs and configures the Brightlayer UI [react-native-auth-workflow](https://www.npmjs.com/package/@brightlayer-ui/react-native-auth-workflow) package to automatically wrap the main application with functions and screens for Login, Registration, Change Password, etc.
```

**Comment by bkarambe** [comment]
@joebochill @daileytj @EvanMcLaughlin-eaton @jeffvg @huayunh Please review.

**Review by daileytj — APPROVED** [review]

**Comment by emclaug2** [comment]
> _**In future, we can rename the templates from blank-typescript to blank, but for now lets use the same**_

Sounds good me 👍

File: package.json [hunk 4]
```diff
@@ -12,19 +12,13 @@
...
-        "start:routing-ios": "cd demo && yarn start:routing-ios",
-        "start:routing-android": "cd demo && yarn start:routing-android",
         "start:routing-typescript-ios": "cd demo && yarn start:routing-typescript-ios",
         "start:routing-typescript-android": "cd demo && yarn start:routing-typescript-android",
-        "start:authentication-ios": "cd demo && yarn start:authentication-ios",
-        "start:authentication-android": "cd demo && yarn start:authentication-android",
         "start:authentication-typescript-ios": "cd demo && yarn start:authentication-typescript-ios",
         "start:authentication-typescript-android": "cd demo && yarn start:authentication-typescript-android",
-        "precommit": "cd demo && bash ./scripts/buildBlank.sh && cd .. && yarn lint && yarn prettier && yarn generate:licenses",
+        "precommit": "cd demo && bash ./scripts/buildBlankTypescript.sh && cd .. && yarn lint && yarn prettier && yarn generate:licenses",
```
Comment (emclaug2) [inline package.json:21]: Where is this file '.scripts/buildBlankTypescript.sh' ?

Comment (daileytj) [reply package.json:21]: It's in the demo project. I'm not sure the demo project has been maintained though.

**Review by jeffvg — APPROVED** [review]

**Comment by jeffvg** [comment]
I put approval on this but for some reason Android didn't load up when I thought it did..
```start:authentication-typescript-android``` in project fails

**Comment by daileytj** [comment]
> I put approval on this but for some reason Android didn't load up when I thought it did.. `start:authentication-typescript-android` in project fails

I don't think this example project has been kept up to date. Shouldn't be hard to get it there, but may have been broken for some time now

**Comment by joebochill** [comment]
@jeffvg  If it works for creating a new project via the CLI, then it should be fine. If you want to log an issue to update the demo project, we can do that (but honestly, we should just make the templates in the new way and then there is no need for a demo project...the templates are runnable in the new configuration).

**Pull request merged**